.sempoconfig/local_config.ini

```diff
@@ -50,31 +50,30 @@ key   = ENC[AES256_GCM,data:9waUglYTJyo+WICunk/OlU24fSNOz2eqBg==,iv:pQlNObIz5jqt
+chain_id                      = ENC[AES256_GCM,data:SDw=,iv:N2DwLkxOq5zXDuatNt5vHsAUUPGYFuKuGaHZMcVOuIA=,tag:36kq4FEgHP9qj5X44q0VSw==,type:str]
+owner_address                 = ENC[AES256_GCM,data:m36RktqxYaxg+UmO01tgztVeTijoq2yTt8Rs24AZlTS0cL86bI02RlQ3,iv:LjIUbUBcV6AE7Xc0PMe9BkC3dx93anLdseAihcXDfyc=,tag:Yo8tcULDcY56n0b3xOK4YQ==,type:str]
+owner_private_key             = ENC[AES256_GCM,data:o3vI0RZfv++IbPm4eeQA6xC170Hx6L8GlPivULOAsU6w0toQcFfUX5qIEchwQ/y8mO5sx2uJ1HalB2AjACi2SA==,iv:kmGJiMYnRSpvmYjcQpQBJXuqiGh/nLZPIk48LB/ALIM=,tag:H5qztiX+0fnwohueEFSX5g==,type:str]
+float_private_key             = ENC[AES256_GCM,data:eHs9KHlEiOWIf8dG0yDUG4GzBUgN/2FhYPjs/42ko55YeiUuC47IDkSk5b+Ur09SsS1hs43tgC/yYhcvjuMsqg==,iv:B68FqVD4lCVAvejgf3thfeDdI98XHOaP8MWdTeBRe5Q=,tag:bZoaL7639obLk9kHdSFAOw==,type:str]
+contract_version              = ENC[AES256_GCM,data:Ww==,iv:1zSRY5I1XiLHavOcN2p17ucqdZOzhZH2Ot7Jcxq7WCk=,tag:DsgzvPVnE5uoheU5suP6Vg==,type:str]
+gas_price_gwei                = ENC[AES256_GCM,data:eg==,iv:4f1P7KKlDZK6SHZJOdFH+NJbMlbgo16awiAmT3X7nIA=,tag:bofceLs6RdqrfpLfHgT+3g==,type:str]
+gas_limit                     = ENC[AES256_GCM,data:SCVr0+iGcYw=,iv:loNC/HimNENt+5sqZSb98OO6Fwj+a4551FuNXBZUONU=,tag:detHP+diFg3GV1ICLvrrcw==,type:str]
+target_transaction_time       = ENC[AES256_GCM,data:GsF+,iv:Du9h4P9lHNPYStA3mwqIi25IVyyQ3XUeB9Mzpe6IwiA=,tag:IWK7GGzjUAGf1qc4QF/HNg==,type:str]
+check_transaction_retries     = ENC[AES256_GCM,data:Kgw=,iv:GMfhbg9sMEK+5eswTNe9+h0pByodKb+4F3quEKNKGA8=,tag:zCR8ZWno4UH4K7UXvHRMQA==,type:str]
+gas_price_provider            = ENC[AES256_GCM,data:XMEcE1LLcvrvtEAHxA9pYnLL+Pr4rkvlr1fK8l77qOcCfoQu,iv:tGahxY0P8kXwmDJJ9N8zGzaT+5lbhDI3XANxf4ls16Y=,tag:OQAD74I29KS7bwa8DandGg==,type:str]
+internal_to_token_ratio       = ENC[AES256_GCM,data:9A==,iv:tL0J/vIJqBe2XXY5JF9ERv1Z1xOm+9PNrF35xzICt8U=,tag:uK1rq3Ml6G1myMd8oo/H/A==,type:str]
+force_eth_disbursement_amount = ENC[AES256_GCM,data:vpOKmAE=,iv:dJcnwcF+Yb4eb9q44irVi+rBbhAo2ksM1gMY07t0mEg=,tag:3Hgxf6UI7n5DXle8OhURdg==,type:str]
+withdraw_to_address           = ENC[AES256_GCM,data:ilR1M9hfvZU7rmxE9mswI+zXcggYLVLY93Io+2hv1PQwTZWVmo2hfTgc,iv:axpKDmIJd4cRewb32POzblmSmg+5gKB8nZ2f4d9pFsA=,tag:H5jO8IAJ1Lb+BOVoNq+/Zg==,type:str]
+system_wallet_target_balance  = ENC[AES256_GCM,data:2/9cuA==,iv:eQdOakRuAw8s+oNWQ51dvdq5tN7MDtPVCU/kbXccWxQ=,tag:7meHERAfJ6r9Y0k91RQa1Q==,type:str]
+system_wallet_topup_threshold = ENC[AES256_GCM,data:jIaD,iv:7WRow6D5ZuyYyZ2F4ctK8vvVKn5Fm3BJ0IvEPfhg1cg=,tag:3wD2OQxBfmjK7GYSoMiq9A==,type:str]
+synchronous_task_timeout      = ENC[AES256_GCM,data:a9Q=,iv:wNUXv3c7Yhj1HLDF4cg+2/ntf2fB3SxeEaoZPUAyRus=,tag:yQfxkF53k73RaSL+SkytMg==,type:str]
+call_timeout                  = ENC[AES256_GCM,data:yw==,iv:v7Kvioqt9kH3V8HWFfnsEx83PS3z8gMMLpdODnlDsWo=,tag:3PJiE6bEXEOKm0iW4bJufg==,type:str]
+http_provider                 = http://127.0.0.1:8545
```

Comment:
it looks like local_config got polluted with non-encrypted config data. We should come up with a way to ensure that doesn't happen on important secrets